strr-web/pages/registry-dashboard.vue

```diff
@@ -192,7 +211,8 @@ const updateFilterOptions = async () => {
   ]
 }
 
-const navigateToDetails = (id: number) => navigateTo(`/application-details/${id.toString()}`)
+const navigateToApplicationDetails = (id: number) => navigateTo(`/application-details/${id.toString()}`)
```

Comment:
Should this be `applicationNumber` and not id? When using string templates with backticks the `.toString()` not needed.